securedrop/tests/test_source.py

```diff
@@ -43,32 +44,6 @@ def test_index(source_app):
         assert 'Return visit' in text
 
 
-def test_all_words_in_wordlist_validate(source_app):
```

Comment:
I'm not sure we want to lose this test. Am I missing a replacement that checks the form validation the same way?

Reply:
To be honest, I wasn't able to figure out which corner case or code path this was really testing. What's a word that would be rejected by the login form? Is there code somewhere that will reject specific words? The only thing I could think of was non-ascii words but the passphrase generator already enforces ascii.

Since the test case runs over every word it's very slow too, so that's why I removed it. If we have a clear idea of what a "non-allowed" word would be, I can add a test and it could be a lot simpler/faster (by just directly validating the words in the word lists, instead of submitting the login form with every word in the list).

Reply:
I think it would have caught a word that somehow didn't pass the [regular expression in `securedrop/source_app/forms/LoginForm`](https://github.com/freedomofpress/securedrop/blob/3d204257c68cfeb35da8d4f04ab906f6d6413e3a/securedrop/source_app/forms.py#L20), but your new checks of the word lists would too, and I don't think it's worth 20 seconds every test run. Let's drop it.